tests/mocks/MockFactory.ts

```diff
@@ -1,8 +1,10 @@
 import { anything, mock, when } from 'ts-mockito';
 import StorageService from '../../services/StorageService';
 
+const DEFAULT_FILE_URL = 's3.amazonaws.com/default-file-url.pdf';
```

Comment:
nit: let's add the leading 'https' to match the actual format of s3 urls even more
```suggestion
const DEFAULT_FILE_URL = 'https://s3.amazonaws.com/default-file-url.pdf';
```